give-double-the-donation.php

```diff
@@ -5,10 +5,10 @@
 use GiveDoubleTheDonation\DoubleTheDonation\AddonServiceProvider;
 
 /**
- * Plugin Name: Give - Double-the-Donation
- * Plugin URI:  https://givewp.com/addons/BOILERPLATE/
- * Description: Easily integrate with the Double-the-Donation employer matching platform.
- * Version:     1.0.0
+ * Plugin Name: Give - Double the Donation
+ * Plugin URI:  https://givewp.com/addons/give-double-the-donation/
+ * Description: Easily integrate with the Double the Donation employer matching platform.
+ * Version:     1.0.0-beta.1
```

Comment:
This needs to be `1.0.0`

Reply:
It was my understanding that we could send the beta to SVN/.ORG and then tag a `1.0.0` with any changes to go live (after approval).